CHANGELOG.md

```diff
@@ -8,13 +8,17 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 -->
 
-The latest published Vello SVG release is [0.3.1](#031-2024-07-29) which was released on 2024-07-29.
-You can find its changes [documented below](#031-2024-07-29).
+The latest published Vello SVG release is [0.5.0](#031-2024-07-29) which was released on 2024-11-16.
```

Comment:
The first link here still has the wrong target, unfortunately. We link to the same place twice.